src/eduid_common/api/decorators.py

```diff
@@ -89,6 +110,10 @@ def __call__(self, f):
         @wraps(f)
         def decorated_function(*args, **kwargs):
             ret = f(*args, **kwargs)
+
+            if isinstance(ret, Response):  # No need to Marshal again, someone else already did that
+                return ret
+
```

Comment:
and, from your comment in the chat, perhaps we'll also need to add the Response class from flask...